### PR TITLE
Add Japanese (ja) locale

### PR DESCRIPTION
**Pull request by ohbarye**
I added `ja` locale for Japanese.
Please review when you have time!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,9 +13,9 @@ package-lock.json
 coverage
 
 # build
-locale
-plugin
+locale/
+plugin/
```
Comment (iamkun) [inline .gitignore]: what's this change for?

Comment (ohbarye) [reply .gitignore]: @iamkun 
To include `src/locale` & `src/plugin` folder. 

I found that changes under `src/locale` are ignored when the `.gitignore` has `locale`. But once it changes to `locale`, it just ignores `locale` and stopped to ignore `src/locale`.

Comment (iamkun) [reply .gitignore]: Sure, got it thanks.

**Pull request merged**